src/t_hash.c

```diff
@@ -2929,8 +2929,8 @@ static void httlGenericCommand(client *c, const char *cmd, long long basetime, i
         return;
 
     /* Verify `numFields` is consistent with number of arguments */
-    if (numFields > (c->argc - numFieldsAt - 1)) {
-        addReplyError(c, "Parameter `numFields` is more than number of arguments");
+    if (numFields != (c->argc - numFieldsAt - 1)) {
+        addReplyError(c, "Parameter `numFields` is different than number of arguments");
```

Comment:
what about:
we don't have F in the json schema.
```suggestion
        addReplyError(c, "Parameter `numfields` must equal number of arguments");
```